### PR TITLE
Remove custom spacy transformers. Add model for multilabel to code.

### PR DESCRIPTION
**Pull request by nsorros**
Removes the need for the custom spacy transformers dependency.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
         'deep-learning': [
             'torch',
             'transformers<=2.0.0',
-            'spacy-transformers @ https://github.com/nsorros/spacy-transformers/tarball/master#egg=spacy-transformers',
+            'spacy-transformers',
```
Comment (aCampello) [inline setup.py:56]: :tada:

File: unpinned_requirements.txt [hunk 2]
```diff
@@ -4,8 +4,8 @@ pandas
 scikit-learn
 torch
 spacy<2.3.0,>=2.2.1
-#spacy-transformers
--e git+https://github.com/nsorros/spacy-transformers#egg=spacy-transformers
+spacy-transformers
+#-e git+https://github.com/nsorros/spacy-transformers#egg=spacy-transformers
```
Comment (aCampello) [inline unpinned_requirements.txt:8]: do you still need it in the unpinned requirements?

Comment (nsorros) [reply unpinned_requirements.txt:8]: i thought i removed that

**Comment by aCampello** [comment]
Seems that the tests failed.

File: unpinned_requirements.txt (hunk 2 above)
Comment (nsorros) [reply unpinned_requirements.txt:8]: it is commented out but will remove

**Comment by aCampello** [comment]
This looks good now.

**Pull request merged**